### PR TITLE
[R4R] Add libutf8proc

### PR DESCRIPTION
**Pull request by jakirkham**
Fixes https://github.com/conda-forge/staged-recipes/issues/6069

Packages Julia's `libutf8proc` originally created by the [Public Software Group]( https://www.public-software-group.org/ ).

### CODE REVIEW

File: recipes/libutf8proc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    utf8proc is a small, clean C library that provides Unicode normalization,
+    case-folding, and other operations for data in the UTF-8 encoding
+  doc_url: https://julialang.org/utf8proc/doc
+  dev_url: https://github.com/JuliaLang/utf8proc
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - mingwandroid
+    - nehaljwani
```
Comment (jakirkham) [inline recipes/libutf8proc/meta.yaml]: Went ahead you, @mingwandroid and @nehaljwani, as maintainers as it looks like you authored the package in AnacondaRecipes. Please let me know if that is ok.

Comment (jakirkham) [reply recipes/libutf8proc/meta.yaml]: Thoughts @mingwandroid and @nehaljwani?

Comment (mingwandroid) [reply recipes/libutf8proc/meta.yaml]: Fine by me.

**Comment by jakirkham** [comment]
Seems reasonable. Have gone ahead and merged the latest `master` into this PR. Please let me know if there is anything else.

**Comment by beckermr** [comment]
LGTM!

**Pull request merged**